sagemaker-python-sdk/mxnet_gluon_cifar10/cifar10.py

```diff
@@ -101,6 +104,9 @@ def train(current_host, hosts, num_cpus, num_gpus, channel_input_dirs, model_dir
             if val_acc > best_accuracy:
                 net.save_params('{}/model-{:0>4}.params'.format(model_dir, epoch))
                 best_accuracy = val_acc
+                logging.info('Saving the model, params and optimizer state in the path /opt/ml/checkpoints')
```

Comment:
nit: please create a variable for the path

Reply:
done